app/src/main/java/org/simple/clinic/editpatient/EditPatientModel.kt

```diff
@@ -19,7 +20,8 @@ data class EditPatientModel(
     val savedPatient: Patient,
     val savedAddress: PatientAddress,
     val savedPhoneNumber: PatientPhoneNumber?,
-    val savedBangladeshNationalId: BusinessId?
+    val savedBangladeshNationalId: BusinessId?,
+    val saveButtonState: ButtonState? = null
```

Comment:
Does this need to be nullable? This reflects the state of the button and it does not need any async work and can be defaulted to the non-progress state, right?

Reply:
Yes it can be, keeping it enabled as the default value

Reply:
Was this done? The type itself is still nullable (`EditPatientStates?`).